src/sentry/static/sentry/app/views/organizationEventsV2/index.tsx

```diff
@@ -19,14 +22,22 @@ import EventDetails from './eventDetails';
 import {ALL_VIEWS} from './data';
 import {getCurrentView} from './utils';
 
-class OrganizationEventsV2 extends React.Component {
-  static propTypes = {
+type Props = {
+  organization: Organization;
+  location: Location;
+  router: ReactRouter.InjectedRouter;
+  // TODO: what is this?
+  params: any;
```

Comment:
This is a parameter object from react-router it has all the `:foo` placeholders as keys and their values.

Reply:
This is now addressed in https://github.com/getsentry/sentry/pull/14345